src/ServiceControl/Operations/ErrorImporter.cs

```diff
@@ -25,95 +58,34 @@ protected override void Setup(FeatureConfigurationContext context)
         {
             var settings = context.Settings.Get<Settings>("ServiceControl.Settings");
 
+            var queueBindings = context.Settings.Get<QueueBindings>();
+            queueBindings.BindReceiving(settings.ErrorQueue);
+
             context.Container.ConfigureComponent<ErrorIngestor>(DependencyLifecycle.SingleInstance);
             context.Container.ConfigureComponent<ErrorPersister>(DependencyLifecycle.SingleInstance);
             context.Container.ConfigureComponent<FailedMessageAnnouncer>(DependencyLifecycle.SingleInstance);
```

Comment:
We should probably remove these from the container as they are internal to the component now.

Reply:
We can't because we depend on the domain events and the `IDocumentSession`. If we had these two available outside of the container, we could do this but I think it would require a bigger componentization effort.

Reply:
But we create these manually inside the component. Who is resolving them from the container?

Reply:
Also, we still allow [resolution of any concrete type with no interfaces](https://github.com/Particular/ServiceControl/blob/master/src/ServiceControl/Bootstrapper.cs#L79) so I think this is redundant.